applications/FluidDynamicsApplication/FluidDynamicsApplication.py

```diff
@@ -1,7 +1,16 @@
 from KratosFluidDynamicsApplication import *
 
+import KratosMultiphysics
 from KratosMultiphysics import _ImportApplication
 application = KratosFluidDynamicsApplication()
 application_name = "KratosFluidDynamicsApplication"
 
 _ImportApplication(application, application_name)
+
+#TODO: We should add a RegistryProcess function to the python_registry.py to put the keyword in front (we don't want the user to do this)
+KratosMultiphysics.Registry.AddItem(
+    "Processes.KratosMultiphysics.FluidDynamicsApplication.ApplyInletProcess",
+    {
+        "ClassName" : "ApplyInletProcess",
+        "ModuleName" : "KratosMultiphysics.FluidDynamicsApplication.apply_inlet_process"
```

Comment:
```suggestion
        "class_name" : "ApplyInletProcess",
        "module_name" : "KratosMultiphysics.FluidDynamicsApplication.apply_inlet_process"
```

please follow our naming convention

Reply:
The point is that we are registering modules using CamelCase (as you'd import them in Python). Then if we were using snake_case for the other items we'd end up with something of this sort `processes.KratosMultiphysics.FluidDynamicsApplication.ApplyInletProcess.module_name`. We considered that it would be better to always use camel case in the keys, meaning that the one above is retrieved as `Processes.KratosMultiphysics.FluidDynamicsApplication.ApplyInletProcess.ModuleName`.

Reply:
ok, up to xou